crates/sui/src/validator_commands.rs

```diff
@@ -689,6 +762,9 @@ impl Display for SuiValidatorCommandResponse {
                     data, serialized_data
                 )?;
             }
+            SuiValidatorCommandResponse::BridgeCommitteeRegistration(response) => {
+                write!(writer, "{}", write_transaction_response(response)?)?;
```

Comment:
nit: I would suggest to just use the standard Display for SuiTransactionBlockResponse, so just do. However, I see that validator's commands output has not been migrated to table format, so probably this is fine for now.
```suggestion
                write!(writer, "{}", response)?;
```